akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -3570,10 +3569,6 @@ final class Source[Out, Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[
   def log(name: String): javadsl.Source[Out, Mat] =
     this.log(name, ConstantFun.javaIdentityFunction[Out], null)
 
-  /**
-   * API MAY CHANGE
-   */
-  @ApiMayChange
```

Comment:
WithContext is in several places

Reply:
@patriknw ping, I think I pushed the last of WithContext now.